doc/examples/encryption.rst

```diff
@@ -787,3 +787,23 @@ An application using Azure credentials would look like, this time using
     coll.insert_one({"encryptedField": "123456789"})
 
 The driver will `acquire an access token <https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token>`_ from the Azure VM.
+
+
+.. _CSFLE rewrap_many_data_key:
+
+
+Re-Wrapping Data Keys with CMK
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :ref:`~pymongo.encryption.ClientEncryption.rewrap_many_data_key` utility method allows you to easily
```

Comment:
Let's avoid using words like "easily" since it could make people feel dumb if they don't understand.